latest/index.bs

```diff
@@ -190,283 +190,291 @@ OME-NGFF tables are compatible with the [AnnData model](https://github.com/scver
+        ├── .zgroup           # The tables group MAY be in root or in another group.
         |
-        ├── varp              # You MAY add a varp group comtainer. The varp group contains sparse arrays that annotate the columns in X.
-        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
-        |   |
-        │   ├── .zgroup
-        |   |
-        |   ├── .zattrs       # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
-        |   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-        |   |                 # `.zattrs` MUST contain `"keys"`, which is an array of the names of the subgroups containing `varp` arrays.
-        |   │
-        │   └── varp_0        # You MAY add a zarr group for each varp array.
-        |       |             # Each varp array MUST have the same number of rows as columns in X.
-        |       |
-        │       ├── .zgroup
-        |       |
-        |       ├── .zattrs   # `.zattrs` MUST contain `"encoding-type"`, which is set to `"csr_matrix"` or `"csc_matrix"` for compressed sparse row and compressed sparse column, respectively.           
-        |       │             # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-        |       |             # `.zattrs` MUST contain `"shape"` which is an array giving the shape of the densified array.
-        |       |
-        |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
-        |       |   |         # `data` MAY be chunked as the user desires.
-        |       |   ├── .zarray
-        |       |   |
-        |       |   ├── 0
-        │       │   |   ...
-        │       |   └── n
-        |       |
-        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
-        |       |   |         # `indices` MAY be chunked as the user desires.
-        |       |   ├── .zarray  # `indices` MUST be an `int` dtype.
-        |       |   |
-        |       |   ├── 0
-        │       │   |   ...
-        │       |   └── n
-        |       |
-        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
-        |           |         # `indptr` MAY be chunked as the user desires.
-        |           ├── .zarray  # `indptr` MUST be an `int` dtype.
-        |           |
-        |           ├── 0
-        │           |   ...
-        │           └── n
+        ├── .zattrs           # `.zattrs` MUST contain "tables", which lists the keys of the subgroups that are tables. In this case, the only table is "my_table".
+                              # hence `.zattrs` should be equal to `{ "tables": [ "table" ] }`.
         |
-        └── uns               # You MAY add a uns containter to store unstructured data.
+        └── table
```

Comment:
Should this match `"my_table"` above? And then also `{ "tables": [ "my_table" ] }` for the `.zattrs`.
I think that would make it clearer that it's a custom name

Reply:
that's a good point, I reverted back to `my_table` everywhere